data/architecture.yml

```diff
@@ -10,7 +10,7 @@ questions:
     -
         question: 'What is the Symfony mail service transport parameter name?'
         answers:
-            - {value: mailer.transport,   correct: true}
+            - {value: swiftmailer.transport,   correct: true}
```

Comment:
hate to be annoying person, also this one https://github.com/ProPheT777/certificationy-data/pull/25